psi4/src/psi4/ccresponse/polar.cc

```diff
@@ -132,8 +132,12 @@ void polar(void) {
         trace[i] = tensor[i][0][0] + tensor[i][1][1] + tensor[i][2][2];
         outfile->Printf("\n\talpha_(%5.3f) = %20.12f a.u.\n", params.omega[i], trace[i] / 3.0);
 
-        /* make sure omega in nm is rounded */
-        long omega_nm_rd = std::lround(omega_nm);
+         std::string omega_nm_rd;
+         if (params.omega[i] == 0.0)
+            omega_nm_rd = "INF ";
+         else
+            /* make sure omega in nm is rounded */
+            omega_nm_rd = std::to_string(std::lround(omega_nm));
```

Comment:
Can we get parenthesis around this if statement?

Reply:
you mean braces?

Reply:
Sorry yes, getting a little late for my poor brain.

Reply:
No problem, formatting has never been my strength :(